tensorflow/training/docker/1.15.3/py36/cu100/Dockerfile.gpu

```diff
@@ -24,7 +24,7 @@ ARG TF_URL=https://tensorflow-aws.s3-us-west-2.amazonaws.com/1.15.3/AmazonLinux/
 
 # The smdebug pipeline relies for following format to perform string replace and trigger DLC pipeline for validating
 # the nightly builds. Therefore, while updating the smdebug version, please ensure that the format is not disturbed.
-ARG SMDEBUG_VERSION=0.9.1
+ARG SMDEBUG_VERSION=0.9.2
```

Comment:
Add this to 1.15.3 py3 as well, which contains the dockerfiles for Python 3.7

Reply:
Done.